tests/e2e/linalg_ext_ops/BUILD.bazel

```diff
@@ -78,96 +170,3 @@ iree_check_single_backend_test_suite(
     ],
     target_backend = "cuda",
 )
-
-iree_cmake_extra_content(
-    content = """
-# Failing on Emscripten: https://github.com/iree-org/iree/issues/12129
-if(NOT EMSCRIPTEN)
-""",
-    inline = True,
-)
-
-iree_check_single_backend_test_suite(
-    name = "check_llvm-cpu_local-task",
-    srcs = enforce_glob(
-        # keep sorted
-        [
-            "attention.mlir",
-            "reverse.mlir",
-            "scan.mlir",
-            "scatter.mlir",
-            "sort.mlir",
-            "top-k.mlir",
-            "winograd_input.mlir",
-            "winograd_output.mlir",
-        ],
-        include = ["*.mlir"],
-    ),
-    driver = "local-task",
-    target_backend = "llvm-cpu",
-)
-
-iree_cmake_extra_content(
-    content = """
-endif()
-""",
-    inline = True,
-)
-
-iree_check_single_backend_test_suite(
-    name = "check_vmvx_local-task",
-    srcs = enforce_glob(
-        # keep sorted
-        [
-            "reverse.mlir",
-            "scan.mlir",
-            "scatter.mlir",
-            "sort.mlir",
-            "top-k.mlir",
-        ],
-        include = ["*.mlir"],
-        exclude = [
-            "attention.mlir",
-            "winograd_input.mlir",
-            "winograd_output.mlir",
-        ],
-    ),
-    driver = "local-task",
-    target_backend = "vmvx",
-)
-
-iree_check_single_backend_test_suite(
-    name = "check_vulkan-spirv_vulkan",
-    srcs = enforce_glob(
-        # keep sorted
-        [
-            "scatter.mlir",
-            # Top-k test disabled due to miscompile on vulkan.
-            #    "top-k.mlir",
-            "sort.mlir",
-            "winograd_input.mlir",
-            "winograd_output.mlir",
-        ],
-        include = ["*.mlir"],
-        exclude = [
-            "attention.mlir",
-            "reverse.mlir",  #TODO(#12415): disabled due to miscompilation on Pixel 6.
```

Comment:
Whoops, need to add this `reverse.mlir` exclusion on Vulkan back (or switch the tests from Pixel 6 to Pixel 8)

Failed here: https://github.com/iree-org/iree/actions/runs/9882624648/job/27296689347#step:6:629